src/components/shared/ListItem/ListItem.tsx

```diff
@@ -0,0 +1,117 @@
+import React, { ReactNode } from 'react';
```

Comment:
As mentioned above, the intention is here that when the time comes, this component also becomes the basis of `ActionListItem` as well as `DraftDecisionItem`